cns/restserver/api.go

```diff
@@ -919,11 +919,15 @@ func (service *HTTPRestService) getNetworkContainerByOrchestratorContext(w http.
 // If received  "GET": Return all NCs in CNS's state file to DNC in order to check if NC refresh is needed
 // If received "POST": Store all the NCs (from the request body that client sent) into CNS's state file
 func (service *HTTPRestService) getOrRefreshNetworkContainers(w http.ResponseWriter, r *http.Request) {
+	logger.Printf("[Azure CNS] NC refresh is initiated by DNC. The state file in CNS will get updated if it is lost.")
+
 	switch r.Method {
 	case http.MethodGet:
+		logger.Printf("[Azure CNS] DNC is attempting to check if the state file in CNS is intact.")
```

Comment:
or, `GET from getOrRefreshNetworkContainers`

Reply:
you don't have to use my example, but it provides just as much info as yours without the potential of lying to me if that's not why this method was called

Reply:
Make sense. Addressed.